app/components/footer_referentiel_component.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class='references_restitution_illettrisme'>
+  <% scope = 'admin.restitutions.cefr.references' %>
+  <div class='row marges-page'>
+    <div class='col'>
+      <div class='row'>
+        <div class='col-auto col-icone-info'>
+          <div><%= svg_tag_base64 'info.svg', class: 'icone' %></div>
+        </div>
+        <div class='col'>
+          <div class='texte-explications'>
+            <%= md t('explications-anlci', scope: scope) %>
+          </div>
+          <div class='col badges-referentiels'>
+            <%= svg_tag_base64 'badges/anlci/profil1-blanc.svg', class: 'badge-references' %>
+            <%= svg_tag_base64 'badges/anlci/profil2-blanc.svg', class: 'badge-references' %>
+            <%= svg_tag_base64 'badges/anlci/profil3-blanc.svg', class: 'badge-references' %>
+            <%= svg_tag_base64 'badges/anlci/profil4-blanc.svg', class: 'badge-references' %>
+            <% if @type == 'cafe-de-la-place' %>
+              <%= svg_tag_base64 'badges/anlci/profil4_h.svg', class: 'badge-references' %>
+              <%= svg_tag_base64 'badges/anlci/profil4_h_plus.svg', class: 'badge-references' %>
+              <%= svg_tag_base64 'badges/anlci/profil4_h_plus_plus.svg', class: 'badge-references' %>
+            <% else %>
+              <%= svg_tag_base64 'badges/anlci/profil4_plus-blanc.svg',
+                                      class: 'badge-references' %>
+              <%= svg_tag_base64 'badges/anlci/profil4_plus_plus-blanc.svg',
+                                      class: 'badge-references' %>
+            <% end %>
+          </div>
```

Comment:
On pourrait peut-être créer un tableau avec les différents profils sur lequel itérer pour éviter cette duplication ?

Reply:
D'ailleurs j'y pense mais on pourrait même éviter d'avoir un if/else en passant en paramètre directement le tableau avec les référentiels qui nous intéressent.